app/react/src/client/testing/index.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  composeStory as originalComposeStory,
+  composeStories as originalComposeStories,
+  setGlobalConfig as originalSetGlobalConfig,
+  CSFExports,
+} from '@storybook/store';
+import { ProjectAnnotations, Args } from '@storybook/csf';
+
+import { render } from '../preview/render';
+import type { Meta, ReactFramework } from '../preview/types-6-0';
+import type { StoriesWithPartialProps, TestingStory } from './types';
+
+/** Function that sets the globalConfig of your storybook. The global config is the preview module of your .storybook folder.
+ *
+ * It should be run a single time, so that your global config (e.g. decorators) is applied to your stories when using `composeStories` or `composeStory`.
+ *
+ * Example:
+ *```jsx
+ * // setup.js (for jest)
+ * import { setGlobalConfig } from '@storybook/react';
+ * import * as globalStorybookConfig from './.storybook/preview';
+ *
+ * setGlobalConfig(globalStorybookConfig);
+ *```
+ *
+ * @param config - e.g. (import * as globalConfig from '../.storybook/preview')
+ */
+export function setGlobalConfig(config: ProjectAnnotations<ReactFramework>) {
+  originalSetGlobalConfig(config);
+}
+
+const defaultGlobalConfig: ProjectAnnotations<ReactFramework> = {
+  render,
+};
+
+/**
+ * Function that will receive a story along with meta (e.g. a default export from a .stories file)
+ * and optionally a globalConfig e.g. (import * from '../.storybook/preview)
```

Comment:
I guess we should tighten the language in these code comments:
 - "component annotations"
 - "project annotations"

Reply:
It was done already, just not pushed!